### PR TITLE
Pioneer 16d gliders

### PR DESCRIPTION
**Pull request by cdobs**
Adds P16d gliders:
CP05MOAS-GL339 D00014
CP05MOAS-GL376 D00014
CP05MOAS-GL379 D00013
CP05MOAS-GL559 D00003
CP05MOAS-PG564 D00004

### CODE REVIEW

File: deployment/CP05MOAS-GL559_Deploy.csv [hunk 1]
```diff
@@ -10,4 +10,10 @@ TS20210517,,TS20210809,,CP05MOAS-GL559-01-ADCPAM000,2,1,2021-05-17T20:34:00,2021
...
\ No newline at end of file
+TS20210517,,TS20210809,,CP05MOAS-GL559-05-PARADM000,2,1,2021-05-17T20:34:00,2021-08-09T19:30:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-PARADM-50145,39.81802,-70.38575,,1000,1000,SS-1,,
+TS20210903,,,,CP05MOAS-GL559-00-ENG000000,3,1,2021-09-03T22:53:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGCON-GLDRCP-00559,39.92405,-70.37323,,1000,1000,SS-2,,
+TS20210903,,,,CP05MOAS-GL559-01-ADCPAM000,3,1,2021-09-03T22:53:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-ADCPAM-43106,39.92405,-70.37323,,1000,1000,SS-2,,
+TS20210903,,,,CP05MOAS-GL559-02-FLORTM000,3,1,2021-09-03T22:53:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-FLORTM-03921,39.92405,-70.37323,,1000,1000,SS-2,,
+TS20210903,,,,CP05MOAS-GL559-03-CTDGVM000,3,1,2021-09-03T22:53:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-CTDGVM-09029,39.92405,-70.37323,,1000,1000,SS-2,,
+TS20210903,,,,CP05MOAS-GL559-04-DOSTAM000,3,1,2021-09-03T22:53:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-DOSTAM-00364,39.92405,-70.37323,,1000,1000,SS-2,,
+TS20210903,,,,CP05MOAS-GL559-05-PARADM000,3,1,2021-09-03T22:53:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-PARADM-50145,39.92405,-70.37323,,1000,1000,SS-2,,
```
Comment (snwhite) [inline deployment/CP05MOAS-GL559_Deploy.csv:19]: Looks good.

File: deployment/CP05MOAS-GL376_Deploy.csv [hunk 2]
```diff
@@ -1,79 +1,85 @@
...
+TS20210131,,AR52,,CP05MOAS-GL376-05-PARADM000,13,1,2021-01-31T18:00:00,2021-03-30T20:29:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-PARADM-50167,39.92755,-71.1765,,1000,1000,SS-1,,
+TS20210903,,,,CP05MOAS-GL376-00-ENG000000,14,1,2021-09-03T22:55:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGCON-GLDRCP-00376,39.9195,-70.37195,,1000,1000,FZ-1,,
+TS20210903,,,,CP05MOAS-GL376-01-ADCPAM000,14,1,2021-09-03T22:55:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-ADCPAM-49696,39.9195,-70.37195,,1000,1000,FZ-1,,
+TS20210903,,,,CP05MOAS-GL376-02-FLORTM000,14,1,2021-09-03T22:55:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-FLORTM-03190,39.9195,-70.37195,,1000,1000,FZ-1,,
+TS20210903,,,,CP05MOAS-GL376-03-CTDGVM000,14,1,2021-09-03T22:55:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-CTDGVM-09083,39.9195,-70.37195,,1000,1000,FZ-1,,
+TS20210903,,,,CP05MOAS-GL376-04-DOSTAM000,14,1,2021-09-03T22:55:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-DOSTAM-00499,39.9195,-70.37195,,1000,1000,FZ-1,,
+TS20210903,,,,CP05MOAS-GL376-05-PARADM000,14,1,2021-09-03T22:55:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-PARADM-50203,39.9195,-70.37195,,1000,1000,FZ-1,,
```
Comment (snwhite) [inline deployment/CP05MOAS-GL376_Deploy.csv:85]: Looks good.

File: deployment/CP05MOAS-GL379_Deploy.csv [hunk 3]
```diff
@@ -70,4 +70,10 @@ TS20210517,,WJ20210721,,CP05MOAS-GL379-01-ADCPAM000,12,1,2021-05-17T20:48:00,202
...
\ No newline at end of file
+TS20210517,,WJ20210721,,CP05MOAS-GL379-05-PARADM000,12,1,2021-05-17T20:48:00,2021-07-22T18:52:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50304,39.81772,-70.39215,,1000,1000,SS-2,,
+TS20210903,,,,CP05MOAS-GL379-00-ENG000000,13,1,2021-09-03T22:51:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGCON-GLDRCP-00379,39.90953,-70.36182,,1000,1000,SS-1,,
+TS20210903,,,,CP05MOAS-GL379-01-ADCPAM000,13,1,2021-09-03T22:51:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-ADCPAM-49016,39.90953,-70.36182,,1000,1000,SS-1,,
+TS20210903,,,,CP05MOAS-GL379-02-FLORTM000,13,1,2021-09-03T22:51:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-FLORTM-04201,39.90953,-70.36182,,1000,1000,SS-1,,
+TS20210903,,,,CP05MOAS-GL379-03-CTDGVM000,13,1,2021-09-03T22:51:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-CTDGVM-09347,39.90953,-70.36182,,1000,1000,SS-1,,
+TS20210903,,,,CP05MOAS-GL379-04-DOSTAM000,13,1,2021-09-03T22:51:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-DOSTAM-00566,39.90953,-70.36182,,1000,1000,SS-1,,
+TS20210903,,,,CP05MOAS-GL379-05-PARADM000,13,1,2021-09-03T22:51:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50304,39.90953,-70.36182,,1000,1000,SS-1,,
```
Comment (snwhite) [inline deployment/CP05MOAS-GL379_Deploy.csv:79]: Looks good.

File: deployment/CP05MOAS-GL339_Deploy.csv [hunk 4]
```diff
@@ -76,4 +76,10 @@ AR52,,AR52,,CP05MOAS-GL339-01-ADCPAM000,13,1,2021-04-02T13:34:00,2021-04-16T15:2
...
\ No newline at end of file
+AR52,,AR52,,CP05MOAS-GL339-05-PARADM000,13,1,2021-04-02T13:34:00,2021-04-16T15:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-PARADM-50242,40.07595,-70.383683,,200,200,EB,,
+TS20210903,,,,CP05MOAS-GL339-00-ENG000000,14,,2021-09-04T00:31:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGCON-GLDRCP-00339,40.06277,-70.35737,,200,200,EB,,
+TS20210903,,,,CP05MOAS-GL339-01-ADCPAM000,14,,2021-09-04T00:31:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-ADCPAM-50865,40.06277,-70.35737,,200,200,EB,,
+TS20210903,,,,CP05MOAS-GL339-02-FLORTM000,14,,2021-09-04T00:31:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-FLORTM-04472,40.06277,-70.35737,,200,200,EB,,
+TS20210903,,,,CP05MOAS-GL339-03-CTDGVM000,14,,2021-09-04T00:31:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-CTDGVM-09338,40.06277,-70.35737,,200,200,EB,,
+TS20210903,,,,CP05MOAS-GL339-04-DOSTAM000,14,,2021-09-04T00:31:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-DOSTAM-00161,40.06277,-70.35737,,200,200,EB,,
+TS20210903,,,,CP05MOAS-GL339-05-PARADM000,14,,2021-09-04T00:31:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-PARADM-50242,40.06277,-70.35737,,200,200,EB,,
```
Comment (snwhite) [inline deployment/CP05MOAS-GL339_Deploy.csv:85]: Looks good.

File: deployment/CP05MOAS-PG564_Deploy.csv [hunk 5]
```diff
@@ -19,4 +19,11 @@ AR52,,TS20210809,,CP05MOAS-PG564-02-DOSTAM000,3,1,2021-04-03T18:08:00,2021-08-09
...
+TS20210903,,,,CP05MOAS-PG564-00-ENG000000,4,1,2021-09-03T19:45:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGCON-GLDRCP-00564,40.34033,-70.6947,,200,200,CNPM/ISPM,,
+TS20210903,,,,CP05MOAS-PG564-01-CTDGVM000,4,1,2021-09-03T19:45:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGINS-CTDGVM-09295,40.34033,-70.6947,,200,200,CNPM/ISPM,,
+TS20210903,,,,CP05MOAS-PG564-02-DOSTAM000,4,1,2021-09-03T19:45:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGINS-DOSTAM-00432,40.34033,-70.6947,,200,200,CNPM/ISPM,,
+TS20210903,,,,CP05MOAS-PG564-03-FLORTM000,4,1,2021-09-03T19:45:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGINS-FLORTM-04089,40.34033,-70.6947,,200,200,CNPM/ISPM,,
+TS20210903,,,,CP05MOAS-PG564-04-FLORTO000,4,1,2021-09-03T19:45:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGINS-FLORTO-01347,40.34033,-70.6947,,200,200,CNPM/ISPM,,
+TS20210903,,,,CP05MOAS-PG564-05-NUTNRM000,4,1,2021-09-03T19:45:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGINS-NUTNRM-00662,40.34033,-70.6947,,200,200,CNPM/ISPM,,
+TS20210903,,,,CP05MOAS-PG564-06-PARADM000,4,1,2021-09-03T19:45:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGINS-PARADM-50202,40.34033,-70.6947,,200,200,CNPM/ISPM,,
```
Comment (snwhite) [inline deployment/CP05MOAS-PG564_Deploy.csv:29]: Looks good.

**Review by snwhite — APPROVED** [review]

File: deployment/CP05MOAS-GL339_Deploy.csv (hunk 4 above)
Comment (rtravis-whoi) [reply deployment/CP05MOAS-GL339_Deploy.csv:85]: looks good

File: deployment/CP05MOAS-GL376_Deploy.csv (hunk 2 above)
Comment (rtravis-whoi) [reply deployment/CP05MOAS-GL376_Deploy.csv:85]: good

File: deployment/CP05MOAS-GL379_Deploy.csv (hunk 3 above)
Comment (rtravis-whoi) [reply deployment/CP05MOAS-GL379_Deploy.csv:79]: good

File: deployment/CP05MOAS-GL559_Deploy.csv (hunk 1 above)
Comment (rtravis-whoi) [reply deployment/CP05MOAS-GL559_Deploy.csv:19]: good

File: deployment/CP05MOAS-PG564_Deploy.csv (hunk 5 above)
Comment (rtravis-whoi) [reply deployment/CP05MOAS-PG564_Deploy.csv:29]: good

**Review by rtravis-whoi — APPROVED** [review]

**Pull request merged**